FirstProject/assets/login.less

```diff
@@ -0,0 +1,73 @@
+//Login.less
+
+body {
+  background-color:#fafafa;
+}
+
+header {
+  background: url('../img/feedback-banner.jpg') center center no-repeat;
+  background-size: cover;
+  height: 600px;
+
+  .grid {
+    height: 100%;
+  }
+}
+
+.login-form {
+  background-color: #ffffff;
+  align-self: center;
+  text-align: center;
+  padding-bottom: 30px;
+
+  .login-form-header {
```

Comment:
Please note that when using BEM (which you seem to be leveraging here), it is unnecessary to nest selectors (within the same block) this way (unless you truly need the added specificity), since - if done right - the targeted DOM structure, along with the selector's specificity is implied through its naming convention (i.e., would you ever reasonably expect the `login-form-header` to be used anywhere else other than inside a `login-form` parent container?).